lib/broccoli/ember-app.js

```diff
@@ -388,12 +385,8 @@ class EmberApp {
     @return
   */
   concatFiles(tree, options) {
-    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.',
-      arguments[2] === SECRET_DEPRECATION_PREVENTION_SYMBOL);
-
-    options.sourceMapConfig = this.options.sourcemaps;
-
-    return concat(tree, options);
+    this.project.ui.writeDeprecateLine('EmberApp.concatFiles() is deprecated. Please use the `broccoli-concat` module directly.');
```

Comment:
@kratiahuja - Yes, but as @Turbo87 pointed out it is already broken (because this PR didn't remove any calls to `this.concatFiles`, meaning that we had already broken the fastboot monkey patch).